Given a temperature in Celsius, convert it to Fahrenheit using the formula F = (C * 1.8) + 32. If the temperature is above 90°F, print “It’s hot outside!” otherwise, print “The temperature is pleasant”.

In [9]:
def func_temp(C):                                     #will take a no. at the place of C
    F = (C*1.8)+32                                    #it is a formula of change C into F
    if F > 90:                                        #F compare to 90, if more than 90  
        return F,"it's hot outside!"                    #print it's hot outside
    else:                                             #if less than 90 
        return F,"The temperature is pleasant"          #print the temperature is pleasant

In [10]:
func_temp(-1)

(30.2, 'The temperature is pleasant')

Test cases:
1. input:30
 output:86, 'The temperature is pleasant'
2. input:-1
 output:30.2, 'The temperature is pleasant' 